hashintel/hash
# Display sidebar on entity and entity type editors

## 🌟 What is the purpose of this PR?




This PR ensures the sidebar is displayed when using the entity and entity type editors.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203312852763953/1203443872976447/f) _(internal)_

## 🔍 What does this change?




See commits.

## 📜 Does this require a change to the docs?



No.

## 🛡 What tests cover this?



None.

## ❓ How to test this?



1. Login to the frontend
2. Go to the entity editor, entity type editor or the new entity type page and verify the sidebar is shown (for example go to `http://localhost:3000/@example-org/entities/<entityId>` and `http://localhost:3000/@example-org/types/entity-type/user`


## 📹 Demo


<img width="1281" alt="image" src="https://user-images.githubusercontent.com/42802102/204510489-3eb8a24c-e318-4485-85dc-dd04bdd67842.png">

<img width="1279" alt="image" src="https://user-images.githubusercontent.com/42802102/204512217-26e1e07e-a4de-4ac6-a0b5-5bb4ecc38e0a.png">
